storage/src/testFixtures/java/tech/pegasys/teku/storage/server/rocksdb/core/MockRocksDbInstance.java

```diff
@@ -120,7 +121,9 @@ public static MockRocksDbInstance createEmpty(final Class<? extends Schema> sche
   public <K, V> Stream<ColumnEntry<K, V>> stream(final RocksDbColumn<K, V> column) {
     assertOpen();
     assertValidColumn(column);
-    return columnData.get(column).entrySet().stream().map(e -> columnEntry(column, e));
+    return columnData.get(column).entrySet().stream()
+        .peek(value -> assertOpen())
```

Comment:
nit: the `Streams.peek()` method to quote its Javadoc: "exists mainly to support debugging", so we might want to stay away from using it in production.

Reply:
yep its just so that the Mock db instance behaves 'similar' to the real one, so I think its ok here...